docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -212,14 +132,60 @@ The AESM service should be up and running. To confirm that, use:
 sudo systemctl status aesmd.service
 ```
 
-### Docker-enabled System
+#### Configuring Intel Quote Provider
 
-An easy way to install and run the AESM service on a [Docker](https://docs.docker.com/engine/)-enabled
+Intel Quote Provider (`libsgx-dcap-default-qpl`) needs to be configured to use
+either the Intel PCS, the PCCS of your cloud service provider, or your
+own PCCS. The configuration file is located at `/etc/sgx_default_qcnl.conf`.
+
+Make sure to always restart the `aesmd.service` after updating the
+configuration, via:
+
+```bash
+sudo systemctl restart aesmd.service
+```
+
+
+#### Intel PCS
+
+Using the Intel PCS is the simplest and most generic way, but it may be less
+reliable than using your cloud service provider (or your own PCCS).
+To use Intel PCS update the `pccs_url` value in `/etc/sgx_default_qcnl.conf`
+to the Intel PCS API URL:
+
+```json
+  //PCCS server address
+  "pccs_url": "https://api.trustedservices.intel.com/sgx/certification/v4/"
+```
+
+#### Cloud service provider
+
+If your cloud service provider provides it's own PCCS, it's recommended to use
+it. Set the `"pccs_url"` value to the respective address:
+
+- Azure: https://global.acccache.azure.net/sgx/certification/v4/
+  - See the [Azure documentation](https://learn.microsoft.com/en-us/azure/security/fundamentals/trusted-hardware-identity-management#how-do-i-use-intel-qpl-with-trusted-hardware-identity-management) for details on configuring the quote provider.
```

Comment:
Collect these long links into its own section and just reference by name (see other docs for examples).